app/src/main/java/com/example/itacademyhw/room/DAO/UserDao.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.itacademyhw.room.DAO
+
+import androidx.room.*
+import com.example.itacademyhw.room.model.UserEntity
+import kotlinx.coroutines.flow.Flow
+
+@Dao
+interface UserDao {
+    @Query("SELECT * FROM userentity ORDER BY last_name ASC")
+    fun getAll(): Flow<List<UserEntity>>
```

Comment:
единичная операция, а не постоянный поток данных, если тебе нужны постоянные изменения бд, то через Flow, в противном случае 
```suggestion
    suspend fun getAll(): List<UserEntity>
```